src/mobot_client/mobot.py

```diff
@@ -181,11 +181,11 @@ def chat_router_coins(self, message, match):
             bonus_coins = BonusCoin.objects.filter(drop=active_drop)
             message_to_send = ChatStrings.COINS_SENT.format(
                 initial_num_sent=active_drop.num_initial_sent(),
-                total=mc_util.pmob2mob(active_drop.initial_pmob_disbursed()),
+                total=active_drop.initial_mob_disbursed(),
             )
             for bonus_coin in bonus_coins:
                 message_to_send += (
-                    f"\n{bonus_coin.number_claimed()} / {bonus_coin.number_available_at_start} - {mc.pmob2mob(bonus_coin.amount_pmob).normalize()} claimed"
+                    f"\n{bonus_coin.number_claimed()} / {bonus_coin.number_available_at_start} - {bonus_coin.amount_mob.normalize()} claimed"
```

Comment:
number_claimed is no longer a function, but rather just a database field. See base.py.

Reply:
👍